src/java/us/kbase/mobu/ModuleBuilder.java

```diff
@@ -343,17 +343,20 @@ private static String getCmdOutput(File workDir, String... cmd) throws Exception
     private static int runTestCommand(TestCommandArgs testArgs, JCommander jc) {
         // Figure out module name.
         // Join together spaced out names with underscores if necessary.
+        int returnCode = 131;
```

Comment:
PR looks good except mysterious 131 exit code. This code would be default one if KB_SDK couldn't run test process at all. Like when token authentication failed. Or file system on host machine doesn't allow to put token/config files in place. What exit code reference do you mean here?